### DISCUSSION TITLE
error to do with solcast integration?

### DISCUSSION THREAD

**Discussion by bert1111** [BJReplay/ha-solcast-solar#185] [General]
Is this a solcast integration error or HA itself?
I see this in the logs after updating solarman integration and rebooting.
I am on the latest version of everything.

`Logger: homeassistant.components.system_health
Bron: components/system_health/__init__.py:89
integratie: Systeemstatus (documentatie, problemen)
Eerst voorgekomen: 15:10:03 (3 gebeurtenissen)
Laatst gelogd: 16:09:47

Error fetching info
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/system_health/__init__.py", line 89, in get_integration_info
    data = await registration.info_callback(hass)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/solcast_solar/system_health.py", line 27, in system_health_info
    used_requests = coordinator.solcast.get_api_used_count()
                    ^^^^^^^^^^^^^^^^^^^
AttributeError: 'mappingproxy' object has no attribute 'solcast'`

**Comment by autoSteve** [comment]
I think this wants investigating, @BJReplay. It's now on my to-do list after getting distracted. (Sorry @bert1111.)

Something is going wrong here. Nothing earth-shatteringly wrong, and not disruptive, but not ideal. What frustrates me is that I have not replicated it yet.

Will convert to an issue.
